src/Psalm/Internal/Codebase/Analyzer.php

```diff
@@ -1167,6 +1167,9 @@ public function addNodeArgument(
         ];
     }
 
+    /**
+     * @param string $reference The symbol name for the reference. Prepend with an asterisk (*) SOME REASON!.
```

Comment:
I've tried to document this, but I didn't work out what the `*` is meant to _mean_!

Reply:
It's a wildcard I think?

Reply:
The only places I see it used are https://github.com/vimeo/psalm/blob/f9fccb2b2dafacc83bf6c43c24d09aa8a589446c/src/Psalm/Internal/Analyzer/ClassAnalyzer.php#L1949 and https://github.com/vimeo/psalm/blob/master/src/Psalm/Internal/Analyzer/ClassAnalyzer.php#L2314 which appears to signify that the reference does not exist in storage. The only place I found that makes use / acknowledges the asterisk is https://github.com/vimeo/psalm/blob/master/src/Psalm/Internal/Codebase/ClassLikes.php#L237 which ignores it.

Reply:
Looking at https://github.com/vimeo/psalm/blob/master/tests/LanguageServer/CompletionTest.php#L656, I think it is meant to mean "doesn't exist", in this unit test, `Ex` of course doesn't exist, so the symbols is `*Ex`